LoopKit/DeviceManager/DeviceLifecycleProgress.swift

```diff
@@ -16,3 +16,8 @@ public protocol DeviceLifecycleProgress {
     var progressState: DeviceLifecycleProgressState { get }
 }
 
+public enum DeviceLifecycleProgressState: Int, CaseIterable, Codable {
```

Comment:
If you are going to make it `Codable`, please change this to a `String` (not an `Int`). This protects against reordering of the cases and is much clearer to anyone using the `Codable` output.